src/commands.ts

```diff
@@ -1048,7 +1048,14 @@ export class CommandCenter {
                 this.focusScm();
                 return;
             }
-            refs = await repository.getRefs();
+
+            const [branches, tags, maxPublic, draftHeads] = await Promise.all([
+                repository.getBranches(),
+                repository.getTags(),
+                repository.getPublicTip(),
+                repository.getDraftHeads(),
+            ]);
+            refs = [...branches, ...tags, ...maxPublic, ...draftHeads];
         }
```

Comment:
Tags, public tip and draft heads are only included when the repo uses branches. They aren't included if it uses bookmarks. Why not?

Reply:
I don't really know how the development using bookmarks goes. I'm guessing it is just like git: you want to start a new thing, first thing you do is create a bookmark. So there wouldn't be any heads without bookmarks, so it does not make much sense to add this to the bookmarks code.

Reply:
It's possible to use bookmarks and still have heads without bookmarks, but it's less useful than when using branches. However, if someone *does* have bookmark-less heads, it feels like those heads should show up in the list. Using bookmarks also doesn't stop someone from wanting to update to a specific tag or the public tip.